.github/workflows/pr.yml

```diff
@@ -12,28 +12,16 @@ on:
       - release/*
 
 env:
-  ENTANDO_OPT_USE_PPL_TAG: "v1.3.0"
-  ENTANDO_OPT_ENVIRONMENTS: "${{ secrets.ENTANDO_OPT_ENVIRONMENTS }}"
+  ENTANDO_OPT_USE_PPL_TAG: "v1.4.1"
```

Comment:
We need to update also post-merge.yml and ga-publication.yml

Reply:
done